iina/de.lproj/Localizable.strings

```diff
@@ -13,141 +13,142 @@
+"alert.mpv_error" = "Fehler %@ (%@) beim setzen der Einstellung %@.";
+"alert.sleep" = "Ruhezustand des Monitors kann nicht verhindert werden!";
 
-"alert.unsupported_audio" = "Externe Audio-Datei nicht unterstützt.";
-"alert.unsupported_sub" = "Externe Untertitel-Datei nicht unterstützt.";
+"alert.unsupported_audio" = "Externe Audio-Datei wird nicht unterstützt.";
+"alert.unsupported_sub" = "Externe Untertiteldatei wird nicht unterstützt.";
 "alert.unsupported_url" = "URL wird nicht unterstützt.";
-"alert.wrong_url_format" = "Falsche URL-Format.";
+"alert.wrong_url_format" = "Falsches URL-Format.";
 
 "alert.error_open" = "Datei oder Stream kann nicht geöffnet werden!";
-"alert.error_finding_file" = "Datei %@ kann nicht gefunden werden!";
-"alert.error_loading_script" = "Das Skript %@ kann nicht geladen werden!";
-"alert.error_saving_file" = "Fehler beim Speichern von %@: %@";
-"alert.error_deleting_file" = "Datei kann nicht gelöscht werden.";
+"alert.error_finding_file" = " Datei %@ kann nicht gefunden werden!";
+"alert.error_loading_script" = "Das Skript %@ konnte nicht geladen werden!";
+"alert.error_saving_file" = "Fehler beim Speichern %@: %@";
+"alert.error_deleting_file" = "Datei konnte nicht gespeichert werden.";
 
-"alert.config.empty_name" = "Dateiname kann nicht leer sein.";
-"alert.config.name_existing" = "Dateiname bereits vorhanden.";
-"alert.config.cannot_create" = "Config-Datei kann nicht erzeugt werden!";
-"alert.config.cannot_write" = "Config-Datei konnte nicht geschrieben werden!";
+"alert.config.empty_name" = "Bitte Namen vergeben.";
+"alert.config.name_existing" = "Name bereits vergeben.";
+"alert.config.cannot_create" = "Konfigurationsdatei konnnte nicht erstellt werden!";
+"alert.config.cannot_write" = "Konfigurationsdatei nicht beschreibbar!";
 
-"alert.filter.incorrect" = "Fehler bei den Filtereinstellungen. Bitte Werte überprüfen.";
+"alert.filter.incorrect" = "Fehler beim erstellen der Filter. Bitte Parameter überprüfen.";
 
-"alert.keybinding_config.error" = "Die Konfigurationsdatei für die Tastaturkürzel \"%@\" konnnte nicht gelesen werden! Standardtastaturkürzel werden benutzt. Bitte die Konfiguration überprüfen.";
+"alert.keybinding_config.error" = "Fehler beim parsen der Tastaturkürzeldatei \"%@\"! Greife auf die Standardeinstellungen zurück. Bitte Einstellungen überprüfen.";
 
-"alert.extra_option.cannot_read" = "Die Benutzerkonfiguration für die erweiterten mpv Optionen kann nicht gelesen werden.";
-"alert.extra_option.empty" = "Schlüsse oder Wert kann nicht leer bleiben!";
-"alert.extra_option.error" = "Fehler beim setzen der Option --%@=%@ Rückgabewert %d. Bitte die die Erweiterten Einstellungen überprüfen Einstellungen > Erweitert.";
-"alert.extra_option.config_folder" = "Konfigurationsverzeichnis konnte nicht ausgewählt werden. \"%@\".";
+"alert.extra_option.cannot_read" = "Die Benutzerdefinierten extra MPV-Optionen können nicht gelesen werden.";
```

Comment:
For proper German grammar "Benutzerdefinierten must be written: "benutzerdefinierten".